Description: Sort the enums and the annotations in the package-tree.html files
Author: Emmanuel Bourg <ebourg@apache.org>
Forwarded: no

--- a/langtools/src/share/classes/com/sun/tools/doclets/internal/toolkit/util/ClassTree.java
+++ b/langtools/src/share/classes/com/sun/tools/doclets/internal/toolkit/util/ClassTree.java
@@ -161,6 +161,15 @@ public class ClassTree {
         for (Iterator<List<ClassDoc>> it = subclasses.values().iterator(); it.hasNext(); ) {
             Collections.sort(it.next());
         }
+
+        Collections.sort(baseEnums);
+        for (Iterator<List<ClassDoc>> it = subEnums.values().iterator(); it.hasNext(); ) {
+            Collections.sort(it.next());
+        }
+        Collections.sort(baseAnnotationTypes);
+        for (Iterator<List<ClassDoc>> it = subAnnotationTypes.values().iterator(); it.hasNext(); ) {
+            Collections.sort(it.next());
+        }
     }
 
     /**
